### PR TITLE
Change route of project status page

### PR DESCRIPTION
**Pull request by ericpgreen2**
This PR:
- Moves the project status page from `/[organization]/[project]/-/deployment` to `/[organization]/[project]`
- Adjusts the redirect logic on `/[organization]`, which should route to either a project's dashboard (if one exists) or to the project's status page
- Edits the browser page titles

Contributes to #2109

### CODE REVIEW

File: web-admin/src/routes/[organization]/+page.svelte [hunk 1]
```diff
@@ -13,7 +13,9 @@
   );
 
   $: if ($projs.data && $projs.data.projects?.length > 0) {
-    goto(`/${$page.params.organization}/${$projs.data.projects[0].name}`);
+    goto(
+      `/${$page.params.organization}/${$projs.data.projects[0].name}/-/redirect`
+    );
```
Comment (begelundmuller) [inline web-admin/src/routes/[organization]/+page.svelte:18]: Any reason not to have the redirect logic in this page?

File: web-admin/src/routes/[organization]/[project]/+page.svelte [hunk 2]
```diff
@@ -1,48 +1,23 @@
...
-  import { goto } from "$app/navigation";
   import { page } from "$app/stores";
-  import Button from "@rilldata/web-common/components/button/Button.svelte";
-  import { useDashboardNames } from "@rilldata/web-common/features/dashboards/selectors";
-  import { runtime } from "@rilldata/web-common/runtime-client/runtime-store";
-  import { createAdminServiceGetProject } from "../../../client";
+  import ProjectDeploymentLogs from "../../../components/projects/ProjectDeploymentLogs.svelte";
+  import ProjectDeploymentStatus from "../../../components/projects/ProjectDeploymentStatus.svelte";
+  import ProjectGithubConnection from "../../../components/projects/ProjectGithubConnection.svelte";
+  import ShareProjectCta from "../../../components/projects/ShareProjectCTA.svelte";
```
Comment (begelundmuller) [inline web-admin/src/routes/[organization]/[project]/+page.svelte:6]: Consider using absolute paths

**Review by begelundmuller — APPROVED** [review]

File: web-admin/src/routes/[organization]/+page.svelte (hunk 1 above)
Comment (ericpgreen2) [reply web-admin/src/routes/[organization]/+page.svelte:18]: The redirect logic depends on calls to a project's runtime, so it's just easier to do within the scope of `/[organization]/[project]/+layout.svelte`, which passes the project's runtime to child components.

File: web-admin/src/routes/[organization]/[project]/+page.svelte (hunk 2 above)
Comment (ericpgreen2) [reply web-admin/src/routes/[organization]/[project]/+page.svelte:6]: Our VSCode settings are: `"typescript.preferences.importModuleSpecifier": "project-relative"`

Think we should change to `shortest` or `non-relative`? Alternatively, Hamilton has proposed SvelteKit aliases.

**Pull request merged**